packages/githublight/style/variables.css

```diff
@@ -0,0 +1,440 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+/*
+The following CSS variables define the main, public API for styling JupyterLab.
+These variables should be used by all plugins wherever possible. In other
+words, plugins should not define custom colors, sizes, etc unless absolutely
+necessary. This enables users to change the visual theme of JupyterLab
+by changing these variables.
+
+Many variables appear in an ordered sequence (0,1,2,3). These sequences
+are designed to work well together, so for example, `--jp-border-color1` should
+be used with `--jp-layout-color1`. The numbers have the following meanings:
+
+* 0: super-primary, reserved for special emphasis
+* 1: primary, most important under normal situations
+* 2: secondary, next most important under normal situations
+* 3: tertiary, next most important under normal situations
+
+Throughout JupyterLab, we are mostly following principles from Google's
+Material Design when selecting colors. We are not, however, following
+all of MD as it is not optimized for dense, information rich UIs.
+
+The name of the variables follow the css patterns described in the css
+jupyterlab developer guide,
+https://jupyterlab.readthedocs.io/en/latest/developer/css.html and github
+vscode themes,
+https://github.com/primer/primitives/blob/main/data/colors/themes/light.ts.
+*/
+@import url('https://fonts.googleapis.com/css2?family=Atkinson+Hyperlegible&display=swap');
```

Comment:
Is there a reason to change the type here?
It feels like, at the bare minimum, this should also be noted in the readme and providing instructions for folks wanting to change the typeface

Reply:
> Is there a reason to change the type here?

I changed it here because it was a request in the latest accessibility meeting. I will note this one in the readme, I think users will still be able to change fonts in the settings.

Reply:
We need to make sure this is optional and customisable. And in any case that this is noted for users